module/dkg/client_mock.go

```diff
@@ -0,0 +1,31 @@
+package dkg
+
+import (
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/flow"
+	model "github.com/onflow/flow-go/model/messages"
+)
+
+type MockClient struct {
```

Comment:
This comment goes for both new mock client types:

The functionality to allow starting up a node without a properly configured machine account is very much intended to be temporary. The only reason we have this is to support the very first mainnet spork with the epoch smart contract. At the beginning of the spork, operators will not have been able to generate their machine account because the smart contracts to do so have not been deployed yet. Therefore, for the duration of the spork, we allow this config to be omitted. For all subsequent sporks, it will be required.

For this PR, I think we should add comments to these components noting that they are temporary and why they exist, with a link to the issue (https://github.com/dapperlabs/flow-go/issues/5619) to deprecate them later on